docker-compose.yml

```diff
@@ -125,7 +125,7 @@ services:
       retries: 3
       test:
         - CMD-SHELL
-        - /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P "$$MSSQL_SA_PASSWORD" -Q "DROP DATABASE IF EXISTS [ibis_testing]; CREATE DATABASE [ibis_testing]"
+        - /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P "$$MSSQL_SA_PASSWORD" -Q "IF DB_ID('ibis_testing') IS NULL BEGIN CREATE DATABASE [ibis_testing] END"
```

Comment:
This command runs periodically, which would lead to strange connection issues. Here, it's still periodic but once the `ibis_testing` database is created it's no longer recreated when the healthcheck runs.